pygmt/helpers/decorators.py

```diff
@@ -40,6 +40,37 @@
         color : str or 1d array
             Select color or pattern for filling of symbols or polygons. Default
             is no fill.""",
+    "I": r"""\
```

Comment:
This docstring uses `r""" """`. I think we should remove the `\` just like `CPT`.
```suggestion
    "I": r"""
```